### PR TITLE
docs: first set of eCHO video links added

### PR DESCRIPTION
**Pull request by Kikiodazie**
This PR adds the first set of eCHO episodes video links added

### CODE REVIEW

File: Documentation/gettingstarted/k8s-install-openshift-okd.rst
Comment (lizrice) [inline Documentation/gettingstarted/k8s-install-openshift-okd.rst]:
```suggestion
To learn more about OpenShift and Cilium, check out `eCHO episode 31: OpenShift Test Environment with Cilium <https://youtu.be/4jwIBS5IzYg?t=945>`__.
```
This would be more consistent with how the other links have been added

**Review by lizrice — COMMENTED** [review]
One suggestion

We have set up Netlify previews now, so with a bit of luck, if you commit this change we'll be able to see a preview. With that preview it will be easier to just double check that all the links are correct before we commit this

It would be great if we can have the videos open in a different tab but I don't think that needs to block this change

File: Documentation/gettingstarted/k8s-install-openshift-okd.rst
Comment (Kikiodazie) [reply Documentation/gettingstarted/k8s-install-openshift-okd.rst]: True, thank you.

**Comment by Kikiodazie** [comment]
> One suggestion
> 
> We have set up Netlify previews now, so with a bit of luck, if you commit this change we'll be able to see a preview. With that preview it will be easier to just double check that all the links are correct before we commit this
> 
> It would be great if we can have the videos open in a different tab but I don't think that needs to block this change

The netlify preview is still failing.

File: Documentation/bpf.rst
Comment (qmonnet) [inline Documentation/bpf.rst]:
```suggestion
To learn more about bpftool, check out `eCHO Episode 11: Exploring 
bpftool <https://www.youtube.com/watch?v=1EOLh3zzWP4&t=650s>`__ with Quentin Monnet, maintainer of bpftool.
```

File: Documentation/concepts/ebpf/lifeofapacket.rst
Comment (qmonnet) [inline Documentation/concepts/ebpf/lifeofapacket.rst]: Let's skip a line

```suggestion

This documentation page overviews the life of a packet from an eBPF datapath perspective 
```

File: Documentation/gettingstarted/clustermesh/clustermesh.rst
Comment (qmonnet) [inline Documentation/gettingstarted/clustermesh/clustermesh.rst]: I'd remove `this video:`, here and everywhere else.

```suggestion
across multiple Kubernetes clusters, check out `eCHO Episode 41: Cilium Clustermesh <https://www.youtube.com/watch?v=VBOONHW65NU&t=342s>`__.
```

File: Documentation/gettingstarted/encryption-wireguard.rst
Comment (qmonnet) [inline Documentation/gettingstarted/encryption-wireguard.rst]: “WireGuard”. It's important. CI should complain about the case here, not sure why it didn't :thinking:. Please also adjust in the video title.

File: Documentation/operations/performance/benchmark.rst
Comment (qmonnet) [inline Documentation/operations/performance/benchmark.rst]: Not sure what's best, “dive deep” or “deep-dive”?

**Review by qmonnet — CHANGES_REQUESTED** [review]
Looks good overall.

Some minor comments/suggestions.

Please make sure that every commit is stand-alone (squash your fixes, e.g. for the bpf.rst page, with the commit that they fix).

We also mentioned using rST admonitions for highlighting those links, we should probably do this in this PR when adding the links.

Finding a way to open external links in new tabs can be a follow-up, though.

**Comment by qmonnet** [comment]
I think the Netlify preview should now work if you rebase your changes on the current `master`.

File: Documentation/gettingstarted/clustermesh/clustermesh.rst
Comment (Kikiodazie) [reply Documentation/gettingstarted/clustermesh/clustermesh.rst]: Will do so. Thanks for the feedback.

File: Documentation/gettingstarted/encryption-wireguard.rst
Comment (Kikiodazie) [reply Documentation/gettingstarted/encryption-wireguard.rst]: Noted. Will do so.

File: Documentation/operations/performance/benchmark.rst
Comment (Kikiodazie) [reply Documentation/operations/performance/benchmark.rst]: I think "dive deep" reads better for the sentence.

File: Documentation/gettingstarted/k8s-install-openshift-okd.rst
Comment (Kikiodazie) [reply Documentation/gettingstarted/k8s-install-openshift-okd.rst]: Screenshot 2022-07-14 at 12 07 20

I left this format in the Talk section of the `bpf.rst` so It would be clear as possible just like other talks listed there. 

Does that work in this case? What do you think?

**Review by sayboras — APPROVED** [review]
LGTM on changes related to clustermesh. Thanks a lot :heavy_check_mark:

File: Documentation/bpf.rst
Comment (lizrice) [reply Documentation/bpf.rst]: Not sure if you meant to resolve this conversation without taking the suggestion? I think it's worth taking to make it consistent

File: Documentation/gettingstarted/kubeproxy-free.rst
Comment (lizrice) [inline Documentation/gettingstarted/kubeproxy-free.rst]: Would be great to start this a couple of seconds later so that the video doesn't start with a cough

File: Documentation/operations/performance/benchmark.rst
Comment (lizrice) [inline Documentation/operations/performance/benchmark.rst]: Could start at 377s so that people can skip me saying hi to Leo!

**Review by lizrice — COMMENTED** [review]
I just checked out the video links, they are all good choices inserted in good places. Suggested small tweaks to the timestamps starting point for a couple of them

File: Documentation/gettingstarted/kubeproxy-free.rst
Comment (Kikiodazie) [reply Documentation/gettingstarted/kubeproxy-free.rst]: About the consistency in eCHO, sorry I missed that one. Will implement all your suggestions right away.

Comment (Kikiodazie) [reply Documentation/gettingstarted/kubeproxy-free.rst]: Does https://www.youtube.com/watch?v=9GBxJMp6UkI&t=980s work?

File: Documentation/operations/performance/benchmark.rst
Comment (Kikiodazie) [reply Documentation/operations/performance/benchmark.rst]: Done!

File: Documentation/gettingstarted/kubeproxy-free.rst
Comment (Kikiodazie) [reply Documentation/gettingstarted/kubeproxy-free.rst]: Hey, @lizrice  Still waiting on your feedback on this before i make the edit.

**Comment by Kikiodazie** [comment]
I have added the admonitions and squash "edits" commits. Please check it out and let me know what you think.

File: Documentation/bpf.rst
Comment (qmonnet) [inline Documentation/bpf.rst]:
```suggestion
  bpftool <https://www.youtube.com/watch?v=1EOLh3zzWP4&t=650s>`__ with Quentin Monnet, maintainer of bpftool.
```

File: Documentation/bpf.rst
Comment (qmonnet) [inline Documentation/bpf.rst]: Please keep the reverse order. If we ever change them we want to do it consistently on the whole page (i.e. other sections too), and this is beyond the scope of this PR.

Looking at the commit history, it looks like you reverted this change, but somehow reintroduced it (inadvertently?) in the last commit.

File: Documentation/bpf.rst
Comment (qmonnet) [inline Documentation/bpf.rst]: Consistency

```suggestion
     with Daniel Borkmann,
     https://www.youtube.com/watch?v=OIyPm6K4ooY&t=308s
```

File: Documentation/concepts/ebpf/lifeofapacket.rst
Comment (qmonnet) [inline Documentation/concepts/ebpf/lifeofapacket.rst]:
```suggestion
  You can also watch a video explanation of this topic that also explores the tools available within Cilium to help 
```
(The video does not focus on “this page” :slightly_smiling_face:)

File: Documentation/concepts/ebpf/lifeofapacket.rst
Comment (qmonnet) [inline Documentation/concepts/ebpf/lifeofapacket.rst]: Please use the full youtube.com URL for consistency with the other links.

File: Documentation/gettingstarted/encryption-wireguard.rst
Comment (qmonnet) [inline Documentation/gettingstarted/encryption-wireguard.rst]:
```suggestion
  WireGuard can encrypt network traffic. 
```

File: Documentation/gettingstarted/clustermesh/clustermesh.rst
Comment (qmonnet) [inline Documentation/gettingstarted/clustermesh/clustermesh.rst]: It's a bit long maybe?

File: Documentation/gettingstarted/k8s-install-openshift-okd.rst
Comment (qmonnet) [inline Documentation/gettingstarted/k8s-install-openshift-okd.rst]: Please use the full youtube.com URL for consistency with the other links.

File: Documentation/intro.rst
Comment (qmonnet) [inline Documentation/intro.rst]:
```suggestion
  If you'd like a video introduction to Cilium, check out this `explanation by Thomas Graf, Co-founder of Cilium <https://www.youtube.com/watch?v=80OYrzS1dCA&t=405s>`__.
```

File: Documentation/bpf.rst
Comment (qmonnet) [inline Documentation/bpf.rst:3243]: Please add a blank line.

```suggestion

.. admonition:: Video
```

**Review by qmonnet — CHANGES_REQUESTED** [review]
Some more adjustments/suggestions.

> Please make sure that every commit is stand-alone (squash your fixes, e.g. for the bpf.rst page, with the commit that they fix).

I see you squashed all fixes together; What I meant instead is to update each commit to add the relevant fixes, so that instead of having a final commit with “minor edits”, your PR contain only clean commits, and the notion of edits/fixes disappears from the Git history.

Similarly, I think you can fold the addition of the `.. admonition` directives into the individual commits.

File: Documentation/bpf.rst
Comment (Kikiodazie) [reply Documentation/bpf.rst]: Thank you.

File: Documentation/concepts/ebpf/lifeofapacket.rst
Comment (Kikiodazie) [reply Documentation/concepts/ebpf/lifeofapacket.rst]: Thank you.

File: Documentation/bpf.rst
Comment (Kikiodazie) [reply Documentation/bpf.rst]: Thank you will do so.

Comment (Kikiodazie) [reply Documentation/bpf.rst]: Done.

File: Documentation/gettingstarted/k8s-install-openshift-okd.rst
Comment (Kikiodazie) [reply Documentation/gettingstarted/k8s-install-openshift-okd.rst]: Implemented the fix. Thank you.

File: Documentation/intro.rst
Comment (Kikiodazie) [reply Documentation/intro.rst]: Implemented the fix. Thank you.

File: Documentation/bpf.rst
Comment (Kikiodazie) [reply Documentation/bpf.rst:3243]: Implemented the fix. Thank you.

File: Documentation/gettingstarted/clustermesh/clustermesh.rst
Comment (Kikiodazie) [reply Documentation/gettingstarted/clustermesh/clustermesh.rst]: Implemented the fix. Thank you

File: Documentation/concepts/ebpf/lifeofapacket.rst
Comment (Kikiodazie) [reply Documentation/concepts/ebpf/lifeofapacket.rst]: Implemented the fix. Thank you

**Review by qmonnet — APPROVED** [review]
Great, thank you so much!

**Comment by qmonnet** [comment]
Feedback from Liz has been addressed, there's reviews from Tam and myself. We can skip the other reviews. Documentation tests have passed, this is ready to merge.

**Pull request merged**
